rnnhearer/data_reader.py

```diff
@@ -64,8 +65,8 @@ def _check_if_contains_samples(self):
     def _create_single_record(
         self, audio_data_file: np.array, label: str
     ) -> Dict[np.array, str]:
-        _, audio_data = wavfile.read(audio_data_file)
-        return {"audio_data": audio_data, "label": label}
+        audio_data = resample(x=wavfile.read(audio_data_file)[1], num=8000)
```

Comment:
Why 8000?

Reply:
Had to cut the sampling in half to make training times bearable. Topic for further discussion in the final model